README.adoc

```diff
@@ -146,9 +145,8 @@ required because the health check service must stay active within the lifetime o
 to report the state.
 
 This class implements the `HealthCheck` interface from MicroProfile Health API. The `call()` method from the
-interface is implemented to report the state of the service by returning a `HealthCheckResponse` object.
-
-The `system` service self-checks. It reads the system property of the host to report a healthy state.
+interface is implemented to report the state of the service by returning a `HealthCheckResponse` object. The `system`
+service self-checks. It reads the system property of the host to report a healthy state.
```

Comment:
What about..."The class also implements the `call()` method from the interface to report the state of the service by returning a `HealthCheckResponse` object."?

Reply:
Fixed

Reply:
This needs a small adjustment, otherwise, the whole paragraph looks a bit odd. How about:
"This class implements the HealthCheck interface from MicroProfile Health API. The class overrides the call() method from the interface to report the state of the service by returning a HealthCheckResponse object."

Reply:
already fixed